modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowNotifications.testcase

```diff
@@ -112,12 +112,46 @@ definition {
 	}
 
 	@description = "Verify that the user can open new user details by notifications"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanOpenNewUserDetailsByNotifications {
+		property test.name.skip.portal.instance = "WorkflowNotifications#CanOpenNewUserDetailsByNotifications";
 
-		// TODO LRQA-70881 CanOpenNewUserDetailsByNotifications pending implementation
+		task ("Give configure Workflow to Single Approver for User") {
```

Comment:
_Given_ instead of _Give_ 
"Given Workflow is configured to Single Approver for User" might work